integration-test/data/api_schemas/facebook-sdk.json

```diff
@@ -0,0 +1,536 @@
+{
+    "FB": {
```

Comment:
this seems to include EVERYTHING, including properties that we don't  care about (e.g. click to load will still work fine). Can we be more precise and only report issues when something related to CTL changes? What I'm worried about is constant alerts from those tests whenever FB updates their library and some internal API changes (e.g. `__globalCallbacks`) or new props are added that we don't care about.

Reply:
My thinking is that it will be less work to tweak the JSON file occasionally to get the test passing, than it will be to attempt to debug breakage on website that our out of date surrogate scripts start breaking. Also personally I would like to know as a point of professional pride if the surrogate scripts I write are broken, or if they are just out of date.

Yea, I agree the included `FB` SDK seems quite large. I don't know which parts of it matter, I'm not sure that is knowable either. But I suppose a simple option might be to only test the YouTube SDK for now?

Reply:
(Maybe also worth noting that @Charlie-belmer currently has a weekly recurring task in Asana to manually check the Facebook SDK for changes. Doing that manually seems like a PITA, when we could automate it like this instead.)

Reply:
Gotcha, yeah, I didn't realize we checked that manually. Automating it, even if it's not perfect, does seem better 👍